.travis.yml

```diff
@@ -1,11 +1,15 @@
 language: csharp
 solution: Singularity/Singularity.sln
 before_install:
-  - wget http://www.builtwithmonogame.com/releases/v3.4/MonoGame.Linux.zip
-  - unzip MonoGame.Linux.zip
-  - find * -type f -exec chmod 777 {} \; # lazy just make all executable
-install:
-  - sudo apt-get install -qq monodevelop libopenal-dev referenceassemblies-pcl ttf-mscorefonts-installer gtk-sharp2; # install needed packages for the monogame sdk
-  - sudo ./generate.sh && yes Y|sudo ./monogame-linux.run; # install monogame sdk
+ - wget http://www.builtwithmonogame.com/releases/v3.4/MonoGame.Linux.zip
+ - unzip MonoGame.Linux.zip
+ - find * -type f -exec chmod 777 {} \; # lazy just make all executable
+ - wget http://www.monogame.net/releases/v3.6/monogame-sdk.run
```

Comment:
Is there a reason we're using 2 different monogame versions?

Reply:
There's actually a bug in automated builds in v 3.4, and they somehow changed the installer of v 3.6 so the build tool is complaining about libraries missing ... so this was a test actually